<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trackify</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>622</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>7</number>
     </property>
     <property name="topMargin">
      <number>7</number>
     </property>
     <property name="rightMargin">
      <number>7</number>
     </property>
     <property name="bottomMargin">
      <number>7</number>
     </property>
     <item row="3" column="4">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
       <property name="text">
        <string>Miejsce nadania:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="9">
      <widget class="QListWidget" name="lwNumery">
       <property name="minimumSize">
        <size>
         <width>315</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>315</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
       <property name="text">
        <string>Data nadania:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5" rowspan="2">
      <widget class="QLabel" name="lMiejsceD">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="4" rowspan="2">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
       <property name="text">
        <string>Miejsce docelowe:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
       <property name="text">
        <string>Przewoźnik:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="lMiejsceN">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="lDataN">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" rowspan="2">
      <widget class="QLabel" name="lKurier">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="5">
      <widget class="QTableWidget" name="twStatusy">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
       <property name="text">
        <string>Dodatkowe informacje:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QLabel" name="lExtra">
       <property name="font">
        <font>
         <family>Lato</family>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Zaaktualizowano:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="8" colspan="3">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1" rowspan="2">
        <widget class="QLabel" name="lNazwa">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="lNumer">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>189</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lNumerP">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Numer przesyłki:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Rodzaj przesyłki:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lRodzaj">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QLabel" name="lNazwaP">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nazwa przesyłki:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="lStatus">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>220</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Status doręczenia:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="5">
      <widget class="QLabel" name="lUp">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="add_parcels"/>
    <addaction name="copy"/>
    <addaction name="edit"/>
    <addaction name="remove"/>
    <addaction name="refresh"/>
    <addaction name="import_parcels"/>
    <addaction name="export_parcels"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="about_app"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="add_parcels"/>
   <addaction name="copy"/>
   <addaction name="edit"/>
   <addaction name="remove"/>
   <addaction name="separator"/>
   <addaction name="refresh"/>
   <addaction name="separator"/>
   <addaction name="import_parcels"/>
   <addaction name="export_parcels"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
  </widget>
  <action name="add_parcels">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/plus.svg</normaloff>:/res/plus.svg</iconset>
   </property>
   <property name="text">
    <string>Dodaj paczkę/paczki</string>
   </property>
   <property name="toolTip">
    <string>Kliknij aby dodać jedną lub wiele paczek!</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="import_parcels">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/folder.svg</normaloff>:/res/folder.svg</iconset>
   </property>
   <property name="text">
    <string>Importuj paczki z pliku</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="export_parcels">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/share.svg</normaloff>:/res/share.svg</iconset>
   </property>
   <property name="text">
    <string>Eksportuj paczki do pliku</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="about_app">
   <property name="text">
    <string>O aplikacji</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Wyjście</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
  </action>
  <action name="refresh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/network.svg</normaloff>:/res/network.svg</iconset>
   </property>
   <property name="text">
    <string>Odśwież</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
  </action>
  <action name="remove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/cancel.svg</normaloff>:/res/cancel.svg</iconset>
   </property>
   <property name="text">
    <string>Usuń paczkę</string>
   </property>
   <property name="toolTip">
    <string>Kliknij aby usunąć paczkę</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/pencil.svg</normaloff>:/res/pencil.svg</iconset>
   </property>
   <property name="text">
    <string>Edytuj paczkę</string>
   </property>
   <property name="toolTip">
    <string>Kliknij aby edytować</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
  </action>
  <action name="copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/copy.svg</normaloff>:/res/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Kopiuj numer</string>
   </property>
   <property name="toolTip">
    <string>Kliknij aby skopiować numer</string>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
